tests/login.test.js

```diff
@@ -0,0 +1,45 @@
+import SignIn from "../pages/SignIn"
+import HomePage from "../pages/HomePage"
+import SignUp from "../pages/SignUp";
+import UserNames from "../datos/UserNames";
+
+describe('Login', () => {
+    UserNames.forEach(({username, password, reason}) => {
+    it(`Should return error when Signing In because ${reason}`, async () => {
+        await SignIn.open('bank')
+        await SignIn.logIn(username, password)
+        await expect(SignIn.signInErrorText).toBeDisplayed()
+    })
+    });
+    it('Should Sign In', async ()=> {
+        await SignIn.open('bank')
+        await SignIn.usernameInput.setValue('jsmith@demo.io')
+        await SignIn.passwordInput.setValue('Demo123!')
+        await SignIn.submit()
+        await expect(HomePage.title).toHaveText('Dashboard')
+        await HomePage.accessUserSettings()
+        await HomePage.logOut()
+    })
+    it('Should take you to Sign In form', async ()=> {
+        await SignIn.open('bank')
+        await SignIn.usernameInput.setValue('federicodelucchi@gmail.com')
```

Comment:
Maybe use your custom `sendText()` method here, that already implements this logic safely (waiting for the element explicitly)